doc/075_scripting.rst

```diff
@@ -245,6 +245,38 @@ are stored in JSON form. Specifying ``--json``  or ``--quiet`` will suppress any
 non-JSON messages the command generates.
 
 
+check
+-----
+
+The ``check`` command uses the JSON lines format with the following message types.
+
+Status
+^^^^^^
+
++--------------------------+------------------------------------------------------------------------------------------------+
+| ``message_type``         | Always "summary"                                                                               |
++--------------------------+------------------------------------------------------------------------------------------------+
+| ``num_errors``           | Number of errors                                                                               |
++--------------------------+------------------------------------------------------------------------------------------------+
+| ``broken_packs``         | Run "restic repair packs ID..." and "restic repair snapshots --forget" to remove damaged files |
++--------------------------+------------------------------------------------------------------------------------------------+
+| ``suggest_repair_index`` | Run "restic repair index"                                                                      |
++--------------------------+------------------------------------------------------------------------------------------------+
+| ``suggest_prune``        | Run "restic prune"                                                                             |
++--------------------------+------------------------------------------------------------------------------------------------+
+
+Error
+^^^^^
+
+These errors are printed on ``stderr``.
+
++----------------------+-------------------------------------------+
+| ``message_type``     | Always "error"                            |
++----------------------+-------------------------------------------+
+| ``message``          | Error message                             |
```

Comment:
Please add a warning that the message `May change in arbitrary ways across restic versions`.

Reply:
Done.

_FYI: I previously just copied it from other commands which do not have this addition._